setup.cfg

```diff
@@ -119,7 +119,6 @@ remote_data_strict = True
 junit_family=xunit1
 filterwarnings =
     error
-    error::sunpy.util.exceptions.SunpyDeprecationWarning
```

Comment:
Do we need to remove this? In fact, did we need this here?

Reply:
The above line errors by default, so there's not much point in this line. I think it's a hangover from when we didn't error by default on everything.